File: m/hello.m
// http://www.midi.co.jp/

#include <stdio.h>
#include "mystr.h"

int
main(
  int argc,
  char * * argv ) {
  id str;
    str = [ MyStr newStr: "Hello World\n" ];
    printf("d: %s\n", [ str name ]);
    [ str print ];
    return (0);
}

File: c/hello.c
#include <stdio.h>
#include <unistd.h>

char sz2[256];
char sz[] = "Hello World!\n";

int
main (
  int argc,
  char * * argv ) {
    while (1) {
        printf(sz);
        sleep(1);
    }
    return (0);
}

